cpp/ps0000/problem solve 0.cpp

```diff
@@ -1,15 +1,16 @@
-#include <iostream>
+#include <iostream> // 2/2
 // اول غلطه كتب @ وهي مفروض #
 // تانى غلطة في كلمه "iostream.h  هي مفروض من غير علامتين تنصيص ومن غير .h

```

Comment:
- [x] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`